### PR TITLE
Add power on api call.

### PR DESCRIPTION
**Pull request by zenhack**
...now that obmd has this, we should add another redirect.

This is built on top of #1020, which should be merged first.

### CODE REVIEW

File: hil/test_common.py [hunk 1]
```diff
@@ -437,7 +438,21 @@ def obmd_cfg():
         be used as a fixture, but not declared here as such; individual
         modules should declare it as a fixture.
         """
-        portno = 8833
+        # Make the port number a function of which worker we're running on,
```
Comment (Izhmash) [inline hil/test_common.py]: Why did this need to get added in this PR?  Did an issue come up?

**Comment by naved001** [comment]
putting the label on this because we want to merge #1020 first.

**Comment by Izhmash** [comment]
#1020 merged, all we need is a rebase now.

**Comment by zenhack** [comment]
Rebased.

**Pull request merged**